src/test/java/hw21.java

```diff
@@ -0,0 +1,36 @@
+import org.example.pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+
+import static org.example.WaitUtils.waitUntilVisibilityOfElementLocatedBy;
+import static org.openqa.selenium.By.*;
+public class hw21 extends BaseTest {
+    String currentPlayListName = "hw17";
+    String newPlayListName = "hw21";
+@Test
+public void renamePlaylist() throws InterruptedException {
+
+
+    LoginPage loginPage = new LoginPage(getDriver());
+    loginPage.login("dzmitry.kimber@testpro.io", "1qAY76Rs");
+
+   WebElement PlayList = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a[class='active']")));
+
+  actions.doubleClick(PlayList);
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)